docs/Releases.md

```diff
@@ -1,3 +1,74 @@
 # Releases Guide
 
-To be written.
\ No newline at end of file
+React Native macOS has 3 types of builds it will publish. that are shared with React Native
+
+1. **Dry Runs** : Only used by CI to test a PR won't break our publish flow
+2. **Nightlies / Canaries** : Published off our main branch with every commit
+3. **Stable Releases** : Published off `*-stable` branches, with a new patch release for every commit.
+
+We use Azure Pipelines for our publish pipeline. The pipeline is defined in `.ado/publish.yml`, and is set to run on pushes to `main` and `*-stable` branches. The pipeline steps differ between stable branches, with the latest as of time of writing (`0.71-stable`) attempting to re-use some of the NodeJS scripts used by our upstream repo React Native in their CircleCI pipeline. 
+
+## Relevant Scripts from React Native Core
+
+There are various nodejs scripts that React Native Core uses to maintain their releases. These have been refactored over time, so I'll be brief and mention the relevant scripts for React Native macOS. For more info on upstream releases, you can take a look at https://reactnative.dev/contributing/release-testing
```

Comment:
```suggestion
There are various scripts that React Native core uses to manage their releases. These have been refactored over time, so I'll be brief and mention the relevant scripts for React Native macOS. For more info on upstream releases, you can take a look at [the documentation](https://reactnative.dev/contributing/release-branch-cut-and-rc0)
```

Reply:
I'm wondering if we should not use the wording "React Native Core" from public messaging. I think only Microsoft folks call it that to distinguish it from React Native macOS / React Native Windows.